### PR TITLE
Add CRT

### PR DESCRIPTION
**Pull request by metanerd**
#### Summary
 Please provide better phrasing :slightly_smiling_face:

### CODE REVIEW

File: company/about-mattermost/list-of-terms.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ COM is short for [Customer Obsession Meeting](https://docs.mattermost.com/proces
 
 Because the term "country" may be either controversial or incorrect when describing a geographic area governed by a state-like political entity we use the term "country/region" to avoid any accidental or implied judgement on the independence of a region. If the term "country" appears to be incorrectly used, when "country/region" is more appropriate, please contact info@mattermost.com.
 
+### CRT
```
Comment (cwarnermm) [inline company/about-mattermost/list-of-terms.md]:
```suggestion
### Collapsed Reply Threads
```

File: company/about-mattermost/list-of-terms.md [hunk 2]
```diff
@@ -38,6 +38,9 @@ COM is short for [Customer Obsession Meeting](https://docs.mattermost.com/proces
...
+The "Collapsed Reply Thread" is a feature to hide replies to a parent message in Mattermost.
```
Comment (cwarnermm) [inline company/about-mattermost/list-of-terms.md:42]: Collapsed Reply Threads (Beta) is a Mattermost Messaging feature offering an enhanced experience for users communicating in threads and replying to messages. When enabled, Collapsed Reply Threads improve users’ ability to process channel content, find, follow, and resume conversations more easily, and keep threaded conversations focused.

**Review by cwarnermm — DISMISSED** [review]
Thanks, @metanerd! This is a great addition to our terms list!

File: company/about-mattermost/list-of-terms.md [hunk 3]
```diff
@@ -38,6 +38,9 @@ COM is short for [Customer Obsession Meeting](https://docs.mattermost.com/proces
 
 Because the term "country" may be either controversial or incorrect when describing a geographic area governed by a state-like political entity we use the term "country/region" to avoid any accidental or implied judgement on the independence of a region. If the term "country" appears to be incorrectly used, when "country/region" is more appropriate, please contact info@mattermost.com.
 
+### Collapsed Reply Threads
+The "Collapsed Reply Thread" is a feature to hide replies to a parent message in Mattermost.
```
Comment (cwarnermm) [inline company/about-mattermost/list-of-terms.md]:
```suggestion
Collapsed Reply Threads (Beta) is a Mattermost Messaging feature offering an enhanced experience for users communicating in threads and replying to messages. When enabled, Collapsed Reply Threads improve users’ ability to process channel content, find, follow, and resume conversations more easily, and keep threaded conversations focused.
```

**Review by cwarnermm — DISMISSED** [review]

**Review by cwarnermm — DISMISSED** [review]

File: company/about-mattermost/list-of-terms.md [hunk 4]
```diff
@@ -38,6 +38,9 @@ COM is short for [Customer Obsession Meeting](https://docs.mattermost.com/proces
 
 Because the term "country" may be either controversial or incorrect when describing a geographic area governed by a state-like political entity we use the term "country/region" to avoid any accidental or implied judgement on the independence of a region. If the term "country" appears to be incorrectly used, when "country/region" is more appropriate, please contact info@mattermost.com.
 
+### Collapsed Reply Threads
```
Comment (esethna) [inline company/about-mattermost/list-of-terms.md]: (CRT)

File: company/about-mattermost/list-of-terms.md [hunk 5]
```diff
@@ -38,6 +38,9 @@ COM is short for [Customer Obsession Meeting](https://docs.mattermost.com/proces
...
+Collapsed Reply Threads (Beta) is a Mattermost Messaging feature offering an enhanced experience for users communicating in threads and replying to messages. When enabled, Collapsed Reply Threads improve users’ ability to process channel content, find, follow, and resume conversations more easily, and keep threaded conversations focused.
```
Comment (esethna) [inline company/about-mattermost/list-of-terms.md]: link to docs on CRT?

**Review by esethna — DISMISSED** [review]

**Comment by esethna** [comment]
Thanks for the PR @metanerd!

File: company/about-mattermost/list-of-terms.md (hunk 5 above)
Comment (cwarnermm) [inline company/about-mattermost/list-of-terms.md]:
```suggestion
Collapsed Reply Threads (CRT) (Beta) is a Mattermost Messaging feature offering an enhanced experience for users communicating in threads and replying to messages. When enabled, Collapsed Reply Threads improve users’ ability to process channel content, find, follow, and resume conversations more easily, and keep threaded conversations focused. See our [Organizing Conversations using Collapsed Reply Threads (Beta)](https://docs.mattermost.com/help/messaging/organizing-conversations.html) product documentation for details. 
```

File: company/about-mattermost/list-of-terms.md (hunk 4 above)
Comment (cwarnermm) [inline company/about-mattermost/list-of-terms.md]:
```suggestion
### Collapsed Reply Threads (CRT)
```

File: company/about-mattermost/list-of-terms.md [hunk 6]
```diff
@@ -38,6 +38,9 @@ COM is short for [Customer Obsession Meeting](https://docs.mattermost.com/proces
 
 Because the term "country" may be either controversial or incorrect when describing a geographic area governed by a state-like political entity we use the term "country/region" to avoid any accidental or implied judgement on the independence of a region. If the term "country" appears to be incorrectly used, when "country/region" is more appropriate, please contact info@mattermost.com.
 
+### Collapsed Reply Threads
+Collapsed Reply Threads (CRT) (Beta) is a Mattermost Messaging feature offering an enhanced experience for users communicating in threads and replying to messages. When enabled, Collapsed Reply Threads improve users’ ability to process channel content, find, follow, and resume conversations more easily, and keep threaded conversations focused. See our [Organizing Conversations using Collapsed Reply Threads (Beta)](https://docs.mattermost.com/help/messaging/organizing-conversations.html) product documentation for details.
```
Comment (cwarnermm) [inline company/about-mattermost/list-of-terms.md]:
```suggestion
Collapsed Reply Threads (CRT) is a Mattermost Messaging feature available in beta offering an enhanced experience for users communicating in threads and replying to messages. When enabled, Collapsed Reply Threads improve users’ ability to process channel content, find, follow, and resume conversations more easily, and keep threaded conversations focused. See our [Organizing Conversations using Collapsed Reply Threads (Beta)](https://docs.mattermost.com/help/messaging/organizing-conversations.html) product documentation for details. 
```

**Pull request merged**